staging/src/k8s.io/csi-translation-lib/plugins/gce_pd.go

```diff
@@ -231,7 +231,7 @@ func (g *gcePersistentDiskCSITranslator) TranslateInTreePVToCSI(pv *v1.Persisten
 		if err != nil {
 			return nil, fmt.Errorf("failed to get region from zones: %v", err)
 		}
-		volID = fmt.Sprintf(volIDZonalFmt, UnspecifiedValue, region, pv.Spec.GCEPersistentDisk.PDName)
+		volID = fmt.Sprintf(volIDRegionalFmt, UnspecifiedValue, region, pv.Spec.GCEPersistentDisk.PDName)
```

Comment:
do we have a unit test for this?

Reply:
The only test we have that covers PV translation is https://github.com/kubernetes/kubernetes/blob/e16c6cc3483989899961c623518a1150e9078495/staging/src/k8s.io/csi-translation-lib/translate_test.go#L47. Since translation from CSI to in-tree doesn't rely on the volume handle, this bug doesn't affect the test. We could add a PD specific test to cover TranslateInTreePVToCSI().

Reply:
Yeah I think a unit test to cover this case would be good

Reply:
Ack - will add a test case.